README.md

```diff
@@ -9,28 +9,34 @@ Digital Archivist might use to export files from S3 once a transfer has been fin
 
 The full export has several steps:
 
-- Download files from S3
+- Download files from S3, and create a directory of files to export
 - Zip the files
-- Encrypt the zip file
 - Upload the encrypted file to a different S3 bucket
 
 You can run the steps separately, or run them together with Docker.
 
-### Step 1: download files
+### Step 1: download and package files
 
 [Configure your AWS credentials][aws-cli-auth] in the `~/.aws/credentials` file. The download step will uses this
```

Comment:
Should read: "The download step will use this..."

Reply:
Fixed.